typescript-declarations/package.json

```diff
@@ -13,19 +13,18 @@
     "carp-studies-core"
   ],
   "dependencies": {
-    "@js-joda/core": "5.2.0",
-    "big.js": "6.1.1"
```

Comment:
I noticed you removed this one, do we not need it?

Reply:
Big JS no longer seems a dependency. I think it was used for the `long` implementation in kotlin standard library. Maybe they now use a different approach. All tests pass ...